### PR TITLE
Created single api demo

### PR DESCRIPTION
**Pull request by AndyMJones**
This PR creates a single package that can be used for demoing browser, Openfin and Electron rather than having to manage 3 different packages. It uses the new api bundle package which links to the 3 specific api packages.

The old demo packages still need to be deleted but i thought i'd wait until these changes had be approved.

### CODE REVIEW

File: packages/api-demo/src/bootstrap.min.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/*!
```
Comment (BenLambertNcl) [inline packages/api-demo/src/bootstrap.min.css:1]: Should this be gitignored?

File: packages/api-demo/src/containerjs-bundle.js [hunk 2]
```diff
@@ -0,0 +1,4263 @@
+(function (global, factory) {
```
Comment (BenLambertNcl) [inline packages/api-demo/src/containerjs-bundle.js]: Same about gitignore

**Review by BenLambertNcl — COMMENTED** [review]
With moving all of the demos into 1, we should be ok to delete `api-specification/src` I think. I would delete the old demos in this PR, but its up to you.

File: packages/api-demo/src/bootstrap.min.css (hunk 1 above)
Comment (AndyMJones) [reply packages/api-demo/src/bootstrap.min.css:1]: Not in its current state but i could bring it in as an npm package if you think that would be better?

Comment (BenLambertNcl) [reply packages/api-demo/src/bootstrap.min.css:1]: No I think as it is now is better :)

**Review by BenLambertNcl — APPROVED** [review]

**Pull request merged**